doc/api/process.md

```diff
@@ -337,7 +337,7 @@ changes:
   rejection or from an synchronous error. Can either be `'uncaughtException'` or
   `'unhandledRejection'`. The latter is only used in conjunction with the
   [`--unhandled-rejections`][] flag set to `strict` or `throw` and
-  an unhandled rejection.
+  an unhandled rejection, or when the entry point is an ES module.
```

Comment:
so this isn't quite true, any ESM load operation could cause this even with `import()` saying entry point might be a bit misleading. How about something like: "when loaded using ESM"? Probably need to workshop phrasing.

Reply:
to be clear CJS can cause uncaughtRejection when loaded through ESM.

Reply:
When the loading mechanism is `import`?

(how can you load CJS through ESM? Do you mean via createRequire?)

Reply:
@ljharb `import './foo.cjs'`

Reply:
ah, right, ofc